### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 6

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#246] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152315064-2425abc0-c2ae-4245-bde6-7b8f98110c83.png)

**Comment by LeuxLello** [comment]
![20220610_115747](https://user-images.githubusercontent.com/104726379/173041658-07e067df-e16b-4c72-800c-98998eddad0a.jpg)

**Comment by Elia-Belli** [comment]
**Osservazione Qualitativa**

-**Iniettività**: f(x) può essere interpretata come "dividere 1 in x parti e sommare 1": dividendo ogni volta per un x diverso si ottiene sempre un numero di parti diverso, sommare 1 trasla solamente il risultato.

-**Suriettività**:
Non esiste alcun numero reale tale che 1/x = 0 in modo che f(x) = 0+1 = 1
